testsuite/E70.py

```diff
@@ -12,6 +12,8 @@
 del a[:]; a.append(42);
 #: E704:1:1
 def f(x): return 2
+#: E704:1:1
+async def f(x): return 2
```

Comment:
I think it's beneficial to trigger both warnings -- it reduces the number of steps to finish refactoring some bad code.

With this patch it immediately shows both warnings, but without this patch you would first see E272, fix it, and then see E704.